test/vault_helpers.go

```diff
@@ -333,6 +333,81 @@ func runVaultWithS3BackendClusterTest(t *testing.T, packerBuildName string, sshU
 	})
 }
 
+// Test the Vault enterprise cluster example by:
+//
+// 1. Copy the code in this repo to a temp folder so tests on the Terraform code can run in parallel without the
+//    state files overwriting each other.
+// 2. Build the AMI in the vault-consul-ami example with the given build name and the enterprise package
+// 3. Deploy that AMI using the example Terraform code
+// 4. SSH to a Vault node and initialize the Vault cluster
+// 5. SSH to each Vault node and unseal it
+// 5. SSH to a Vault node and make sure you can communicate with the nodes via Consul-managed DNS
```

Comment:
`5` is repeated twice (if you copy/pasted this comment block, might be a bug in the original!). Also, add another step here: check enterprise version of Vault got installed correctly.